### ISSUE TITLE
Chart style

### ISSUE THREAD

**Issue by jfrankl** [azavea/climate-change-lab#89]
### Colors

Primary color: #1E8CEC ![image](https://cloud.githubusercontent.com/assets/1809908/18894723/9293e450-84e2-11e6-939c-f91b79590d7c.png)
Trend line: #D44A34 ![image](https://cloud.githubusercontent.com/assets/1809908/18894736/a15d7dc0-84e2-11e6-9e82-842f681e0e56.png)
Max block: #F3B32F ![image](https://cloud.githubusercontent.com/assets/1809908/18894748/b42e9d94-84e2-11e6-8f90-7732dc12c839.png)
Min block: #41424D ![image](https://cloud.githubusercontent.com/assets/1809908/18894751/bfef1b4a-84e2-11e6-8ee9-338a9f4fa9c2.png)
Both block: #E5A0C0 ![image](https://cloud.githubusercontent.com/assets/1809908/18894759/d0048c54-84e2-11e6-93d1-030d9c207e47.png)
Band: #EFF4F7 ![image](https://cloud.githubusercontent.com/assets/1809908/18894771/e344daee-84e2-11e6-920a-e4d8f65ce482.png)
Grid: #757575 

### Weights & Opacity
- Min, max, and both blocks have 0.3 opacity.
- The grid lines displayed as red in the below image should have 0.5 width (for color, see `Grid` above)
![image](https://cloud.githubusercontent.com/assets/1809908/18894836/438c7ea2-84e3-11e6-966e-9aa0b33cf320.png)
- The grid lines displayed as red in the below image should have 1.0 width (for color, see `Grid` above)
![image](https://cloud.githubusercontent.com/assets/1809908/18894854/6410da88-84e3-11e6-956d-ef4d27133f8b.png)

### More mockups

The following mockups show what the chart should look like in the application.

![image](https://cloud.githubusercontent.com/assets/1809908/18894651/1c6641c4-84e2-11e6-9c05-ce8e151b8ee6.png)

![image](https://cloud.githubusercontent.com/assets/1809908/18894669/3ef2279e-84e2-11e6-8fe3-8980e3036c5c.png)

![image](https://cloud.githubusercontent.com/assets/1809908/18894662/27eba4d0-84e2-11e6-9f09-99cf8d7c10e5.png)

cc @jbranigan

**Comment by fungjj92** [comment]
Assigning a different and random color to the 'both' bar is visually confusing to me. Is there an issue with using the color resulting form an overlap of the two bar colors (yellow, grey)? The overlap is how it is, now.

Clarifying -- the big red gridlines are red just for visibility/clarity but should really be 757575?

**Comment by jfrankl** [comment]
I'm not sure which is better. I just created a mockup of the different colors and overlapping colors. I also included a mockup that uses a pattern fill that includes both colors, which I think is the most clear. I will defer to you and @jbranigan on which one you want to go with.

![min-max-options](https://cloud.githubusercontent.com/assets/1809908/18919673/10d55472-856d-11e6-9f6a-3f50709ab8e7.png)

Clarification: yes, the gridlines should be #757575.
